src/components/utils/error/ClientInfo.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * A component used to display details about the user and user agent that may
+ * be helpful when troubleshooting errors.
+ */
```

Comment:
I extracted this from `ErrorHandler.js`. That seemed like the most reasonable thing to do to me. I'm curious to know what others think, though.